### ISSUE TITLE
订阅，关注 topic

### ISSUE THREAD

**Issue by xu-song** [xu-song/TODO#1]
github不能follow group和topic。。很猥琐。只能自己收录咯。

## github topic

| 机器学习 | text | vision | speech | 工具 | 算法 数据结构 | 系统 | math |
|----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------|---------------|----------------------------------------------------------------|------------------------------------------------------------------|
| [seq2seq](https://github.com/topics/seq2seq) | [text-classification](https://github.com/topics/text-classification) | [imagenet](https://github.com/topics/imagenet) | [asr](https://github.com/topics/speech-recognition)  [2](https://github.com/topics/speech-to-text) [3](https://github.com/topics/automatic-speech-recognition) | [pytorch](https://github.com/topics/pytorch) |  | [self-driving-car](https://github.com/topics/self-driving-car) | [learn math fast](https://github.com/llSourcell/learn_math_fast) |
| [attention-model](https://github.com/topics/attention-model)[2](https://github.com/topics/attention-mechanism) | [knowledge-graph](https://github.com/topics/knowledge-graph) | [detection](https://github.com/topics/detection)[2](https://github.com/topics/object-detection) | [kaldi](https://github.com/topics/kaldi) | [keras](https://github.com/topics/keras) |  | [ocr](https://github.com/topics/ocr) |  |
|  | [knowledge-base](https://github.com/topics/knowledge-base) | [segmentation](https://github.com/topics/instance-segmentation) | [tts](https://github.com/topics/text-to-speech) [2](https://github.com/topics/speech-synthesis) [3](https://github.com/topics/tts) |  |  | [blockchain](https://github.com/topics/blockchain) |  |
|  | [machine-translation](https://github.com/topics/machine-translation) | [image-recognition](https://github.com/topics/image-recognition) | [fst](https://github.com/topics/fst) [openfst](https://github.com/topics/openfst) [wfst](https://github.com/topics/wfst) |  |  |  |  |
| [reinforcement-learning](https://github.com/topics/reinforcement-learning) | [sentiment-analysis](https://github.com/topics/sentiment-analysis) | [action-recognition](https://github.com/topics/action-recognition) |  |  |  | [chatterbot](https://github.com/topics/chatterbot) |  |
| [capsnet](https://github.com/topics/capsnet) | [conversation](https://github.com/topics/conversation) | [action-detection](https://github.com/topics/action-detection) |  |  |  |  |  |
| [gan](https://github.com/topics/gan) [2](https://github.com/topics/generative-adversarial-network) | [chatbot](https://github.com/topics/chatbot) - [2](https://github.com/topics/chatterbot) | [video-understanding](https://github.com/topics/video-understanding) |  |  |  |  |  |
|  | [language model](https://github.com/topics/language-model) | [video-classification](https://github.com/topics/video-classification) |  |  |  |  |  |
| [fasterrcnn](https://github.com/topics/fasterrcnn) | [nmt]() | [video-segmentation](https://github.com/topics/video-segmentation) |  |  |  |  |  |
| [mask-rcnn](https://github.com/topics/mask-rcnn) |  | [video-captioning](https://github.com/topics/video-captioning) |  |  |  |  |  |
|  |  | [super-resolution](https://github.com/topics/super-resolution) |  |  |  |  |  |

## github group

- [pytorch](https://github.com/pytorch)
- [facebookresearch](https://github.com/facebookresearch)
- [facebook](https://github.com/facebook)
- [tensorflow](https://github.com/tensorflow)
- [google](https://github.com/google)
- [openai](https://github.com/openai)
- [scikit-learn](https://github.com/scikit-learn)
- [tencent](https://github.com/tencent)
- [baidu-research](https://github.com/baidu-research) 落后了

## mooc

| mooc |   |   |   |   |
|------|---|---|---|---|
|  [medium](https://medium.com/)    |   |   |   |   |
|      |   |   |   |   |
|      |   |   |   |   |
|      |   |   |   |   |

## 其他 

awesome list

**Comment by xu-song** [comment]
可以作为首页、导航页

**Comment by xu-song** [comment]
博客

- 腾讯http://www.flickering.cn/
- [我爱自然语言处理](http://www.52nlp.cn/)
- [licstar的博客](http://licstar.net/)

**Comment by xu-song** [comment]
# 电子书资源站
- [ireadweek](https://www.google.com.hk/search?q=site:www.ireadweek.com+s) 有大量kindle电子书
- [scilib.org](https://www.google.com.hk/search?q=site:down.scilib.org+pdf)  大量扫描版电子书
- [子乌](http://book.zi5.me/)  杂书比较多
- [readfree.me](http://readfree.me/) 大量kindle电子书，屏蔽搜索引擎。暂停注册
- kindle114  挂了
- [北大](https://www.google.com.hk/search?q=site:pku.edu.cn+pdf)
- [复旦](https://www.google.com.hk/search?q=site:fudan.edu.cn+pdf)
- [浙大](https://www.google.com.hk/search?q=site:zju.edu.cn+pdf)
- [北师](https://www.google.com.hk/search?q=site:bnu.edu.cn+pdf)
- [新浪微盘](https://www.google.com.hk/search?q=site%3Avdisk.weibo.com+pdf) 不支持内部搜索，貌似是被禁了

**Comment by xu-song** [comment]
aa
